src/main/java/com/dumbdogdiner/stickyapi/annotation/OnHold.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.annotation;
+
+import java.lang.annotation.Documented;
+
+/**
+ * Denotes a feature or class that is in-progress but has had development halted
+ */
+@Documented
+public @interface OnHold {
```

Comment:
Why? If it's work in progress, leave it in a development branch.

Reply:
Still useful to have, to note things in development with a pause

Reply:
Okay, but please don't add annotations like this without checking beforehand.

Reply:
Preferably keep features to their own branches too.